src/services/pullRequestService.ts

```diff
@@ -17,30 +17,46 @@ class PullRequestService {
       return false
     }
 
-    if (this.isActualSemVerHigher(pullRequest, internalContext)) {
-      warning('Version upgrade is higher than configured. Will not merge PR')
+    if (!this.isUpgradeAllowed(pullRequest, internalContext)) {
+      warning('Upgrade is not allowed. Will not merge PR')
       return false
     }
 
     return true
   }
 
-  private isActualSemVerHigher (pullRequest: PullRequestModel, internalContext: InternalContext) {
-    const semVer = this.determineSemVerUpgrade(pullRequest)
+  private isUpgradeAllowed (pullRequest: PullRequestModel, internalContext: InternalContext) {
+    const proposedUpgradeSection = this.getProposedUpgradeFromPrTitle(pullRequest)
 
-    if (!semVer) {
-      return true
+    if (!proposedUpgradeSection) {
+      warning('PR title does not contain enough information about the affected versions')
+      return false
+    }
+
+    const proposedSemVerUpgrade = this.determineSemVerUpgrade(proposedUpgradeSection)
+
+    if (!proposedSemVerUpgrade) {
+      warning('Cannot determine the version upgrade bump')
+      return false
     }
 
-    return semVer > internalContext.input.semVerLimit
+    return internalContext.input.semVerLimit >= proposedSemVerUpgrade
+  }
+
+  private getProposedUpgradeFromPrTitle (pullRequest: PullRequestModel) : string | undefined {
+    const regex = /\sfrom\s(?<proposedUpgrade>.+)/iu
```

Comment:
What is `<proposedUpgrade>`?

Reply:
It's just an identifier for that section in the regex match. It will basically be "v1 to v1.1", for example